example/image-classification/common/fit.py

```diff
@@ -117,6 +117,8 @@ def add_fit_args(parser):
                        help='load the model on an epoch using the model-load-prefix')
     train.add_argument('--top-k', type=int, default=0,
                        help='report the top-k accuracy. 0 means no report.')
+    train.add_argument('--loss', type=str,
+                       help='report the cross-entropy or nll-loss. ce means cross-entropy, nll-loss means likelihood loss')
```

Comment:
This seems a little confusing to me, as in what's the actual expected loss string.
Maybe something like:
"report the loss. Options are ce (cross-entropy) or nll-loss (likelihood loss)
?

?

Reply:
@piiswrong @cjolivier01 Thanks for all your comments, and sorry for the confusion.

Let me try to explain:
1. My purpose is to report the loss value during training, this is because the loss value trends is helpful to monitor the convergence trend.
2. Per my understanding is, the cross-entropy loss and (negative) likelihood loss are most common used for softmax output, and therefore I choose reporting either cross-entropy or negative likelihood loss as long as the output layer is the softmax.

The current implementation of my PR is add a string-type argument, that is the "--loss", and the corresponding type loss will be reported accordingly.

Meanwhile, my understanding to Eric's comment is, it be better to set the the argument as a list including all the matching loss type (['ce', 'nll-loss'] for softmax), and give the user to decide which loss value will be used. 

May I know which method is acceptable? I'll then update my implementation accordingly.
Thanks for your time.

Reply:
@cjolivier01 As my repo is out-of-date to the current mxnet master, I close this PR and submit a new one https://github.com/apache/incubator-mxnet/pull/9805, in which I address your review comments and provide a more detailed description of the PR. Could you please check the PR9805 accordingly? Thanks for your time.